### ISSUE TITLE
[FEATURE] Memoria en problemas CSharp

### ISSUE THREAD

**Issue by licgerman** [omegaup/omegaup#1787]
Apenas ayer resolvieron un issue relacionado con problemas viejos NO-KAREL que no aparecían disponibles para evaluación con CSharp. Este issue ya fue cerrado pero me parece que ahora hay otro problema.

Algunos problemas viejos que no tenían en cuenta a CSharp estaban considerando solamente 4mb de memoria. En CSharp al parecer se requiere más por la librería System.  ¿Es posible modificar TODOS los problemas de CSharp para que tengan la memoria mínima suficiente para que funcione adecuadamente?

**Comment by lhchavez** [comment]
no, pero tal vez podemos hacer lo mismo que se hace para Java, que es dar 14-16MB de memoria extra para el runtime únicamente para envíos hechos en ese lenguaje.

**Comment by licgerman** [comment]
Ok. Gracias. Quedo al pendiente.
